### PR TITLE
Pikcube.OrangeJuiceModMaker version 0.9.6

### PR DESCRIPTION
**Pull request by pikcube**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/98139)

### CODE REVIEW

**Comment by stephengillie** [comment]
Hi @pikcube,

The program is prompting for install location:
image

Is there an installer switch to specify the location and bypass the prompt?

**Comment by Trenly** [comment]
@stephengillie - Try `/ALLUSERS` for machine scope and `/CURRENTUSER` for user scope, since that is an inno installer

**Comment by pikcube** [comment]
And that's what I get for not remembering to uninstall my app before testing it. @Trenly is spot on, that's what needed fixed. I tested it on my local copy and I just fixed it on GitHub so we'll see if it runs now.

**Comment by pikcube** [comment]
I didn't find an option for a scope switch in winget so I added /AllUsers as a Custom switch, but if there is a way to change that I can.

File: manifests/p/Pikcube/OrangeJuiceModMaker/0.9.6/Pikcube.OrangeJuiceModMaker.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+PackageIdentifier: Pikcube.OrangeJuiceModMaker
+PackageVersion: 0.9.6
+Installers:
+- Architecture: x64
+  InstallerType: inno
+  InstallerUrl: https://github.com/pikcube/OrangeJuiceModMaker/releases/download/v0.9.6/OJSetup.exe
+  InstallerSha256: 545112664F22CD6D2A03AD8643D641241EE8FC7CD34823E4A6AA46BC9D75A4C6
+  InstallerSwitches:
+    Silent: /SILENT
+    Custom: /AllUsers
```
Comment (Trenly) [inline manifests/p/Pikcube/OrangeJuiceModMaker/0.9.6/Pikcube.OrangeJuiceModMaker.installer.yaml:13]:
```suggestion
- Architecture: x64
  InstallerType: inno
  InstallerUrl: https://github.com/pikcube/OrangeJuiceModMaker/releases/download/v0.9.6/OJSetup.exe
  InstallerSha256: 545112664F22CD6D2A03AD8643D641241EE8FC7CD34823E4A6AA46BC9D75A4C6
  InstallerSwitches:
    Silent: /SILENT
    Custom: /ALLUSERS
  Scope: machine
- Architecture: x64
  InstallerType: inno
  InstallerUrl: https://github.com/pikcube/OrangeJuiceModMaker/releases/download/v0.9.6/OJSetup.exe
  InstallerSha256: 545112664F22CD6D2A03AD8643D641241EE8FC7CD34823E4A6AA46BC9D75A4C6
  InstallerSwitches:
    Silent: /SILENT
    Custom: /CURRENTUSER
  Scope: user
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by Trenly** [comment]
Being inno, the silent switch should already be known by the cli; Using the custom switch is correct for the scope, but the actual Scope also needs to be specified for the installer. Adding a second installer node (as shown in the suggestion) can be used to account for both the user scope and machine scope

**Comment by pikcube** [comment]
> Being inno, the silent switch should already be known by the cli; Using the custom switch is correct for the scope, but the actual Scope also needs to be specified for the installer. Adding a second installer node (as shown in the suggestion) can be used to account for both the user scope and machine scope

That was exactly what I was trying to do, thank you!

**Review by stephengillie — APPROVED** [review]

**Pull request merged**